### DISCUSSION TITLE
How to setup VS Code

### DISCUSSION THREAD

**Discussion by ajlakanen** [sumatrapdfreader/sumatrapdf#2864] [Q&A]
Visual Studio Code is my current favorite editor (previously it was Sublime Text, and Sumatra worked well with it). Are there up-to-date instructions on how to set up Sumatra with Code? Sumatra is so much better than Edge... I hope I don't have to go back to Sublime since I have started to move all my workflow into Code.

**Comment by GitHubRulesOK** [comment]
there are some changes that seem to affect recent vscode support. So old instructions may need to be avoided ? 

Latest install for Workshop should be here https://marketplace.visualstudio.com/items?itemName=James-Yu.latex-workshop
https://github.com/James-Yu/LaTeX-Workshop/wiki/View#using-synctex-with-an-external-viewer

The process is "relatively" simple

VScode to build a PDF using LaTeX requires LaTeX editor support outside SumatraPDF remit since as a pdf viewer it simply is tasked to open the PDF for pre-view. 
The ability to reverse call an editors line number relies on the LaTeX extender adding a synctex index which currently has one open issue when installed ***and*** using syctex.gz compression (its the installed and compress is an issue)
So there are hints by others how they configure say LaTeX Workshop to link the editor to preview and vice versa. but do ensure they are very recent set-ups due to workshop and VSCode.exe changes.
